### ISSUE TITLE
Delayed sound with bluetooth earbuds

### ISSUE THREAD

**Issue by ricocesar85** [jellyfin/jellyfin#2684]
Hi,

I did a fresh install in Ubuntu server 19.10, therefore I added some videos. When I have tried to got the sound throught bluetooth earbuds, I have noticed a delay between the sound and the video, the delay is almost one second.

Best regards,

**Comment by JustAMan** [comment]
Is this by using the Android app by any chance?

**Comment by ricocesar85** [comment]
Yes, It is. Regards, Enviado desde Correo para Windows 10 De: VasilyEnviado: viernes, 3 de abril de 2020 9:35 a. m.Para: jellyfin/jellyfinCC: ricocesar85; AuthorAsunto: Re: [jellyfin/jellyfin] [bug] Delayed sound with bluetooth earbuds (#2684) Is this by using the Android app by any chance?—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
